@types/jsb.d.ts

```diff
@@ -65,7 +66,7 @@ declare namespace jsb {
     export let onMouseUp: MouseEventCallback | undefined;
     export let onMouseWheel: MouseWheelEventCallback | undefined;
 
-    type TouchEventCallback = (touchList: TouchList) =>  void;
+    type TouchEventCallback = (touchList: TouchList, windowId: number) =>  void;
```

Comment:
What if multiple touches in `touchList` come from defferent windowId?

Reply:
This situation is not surported for now.

Reply:
If multiple touches  come from defferent window, `windowId` should be added to struct Touch.